src/transactions/utils/format.js

```diff
@@ -12,6 +12,37 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import bignum from 'browserify-bignum';
+import { MAX_TRANSACTION_AMOUNT } from 'lisk-constants';
+import { FIXED_POINT } from '../constants';
+
+const getDecimalPlaces = amount => (amount.split('.')[1] || '').length;
+const isGreaterThanMaxTransactionAmount = amount =>
+	amount.cmp(MAX_TRANSACTION_AMOUNT) > 0;
+
+export const convertBeddowsToLSK = beddowsAmount => {
```

Comment:
maybe check if the input is string or not?
Although it is on a protocol, but it might be ambiguous from the nature of amount.